embulk-input-mysql/src/test/java/org/embulk/input/mysql/BasicTest.java

```diff
@@ -41,6 +42,7 @@ private static String readResource(String fileName)
     @Before
     public void setup()
     {
+        TimeZone.setDefault(TimeZone.getTimeZone("Europe/Helsinki")); // +200
```

Comment:
TravisCI has been using `gradle check` command. The gradle reads the following test settings that configure the time zone. Doesn't the setting work for you?
https://github.com/embulk/embulk-input-jdbc/blob/master/build.gradle#L110